README.md

```diff
@@ -124,6 +125,26 @@ docker-compose stop config-editor && docker-compose rm config-editor
 
 For full documentation about the config editor, see the docs for the product at [github.com/cdr/code-server][code-server].
 
+#### Migration from venv
+
+If you are currently running in a venv it is easy to migrate to a docker container by copying the existing venv data to a docker mapped persistent volume.
+
+e.g
+
+```
+cp -r /home/octoprint/.octoprint/. /docker/octoprint
+```
+
```

Comment:
the current documentation all encourages the use of docker volumes instead of host mounted volumes.  Your example is the first place in the docs where a host-mounted volume is used, so it makes sense that your example code should include a command to make that directory.

Technically, there's no reason to even do a copy if you're using host-mounted volumes, since there's no reason to move the files.

Either way, it should probably include a bit of an example for a `docker-compose.yml` file.